### ISSUE TITLE
Font color overrides tree-view and statusbar colors

### ISSUE THREAD

**Issue by richarddewit** [braver/isotope-ui#50]
Highlights like file added/modified in the tree-view, linter status and lines added/removed in statusbar are all plain white (#FAFAFA).

It's caused by [line 188 of config.less](https://github.com/braver/isotope-ui/blob/6162b9b6de2a1260b93dd6be4f31d83254a966f5/styles/config.less#L188) which overrides every other text color sadly.

**Comment by braver** [comment]
Yes, that's kinda intentional to create a reliable contrast with background images. I'm open for suggestions though!
